macroid-core/src/main/scala/macroid/contrib/ExtraTweaks.scala

```diff
@@ -20,9 +20,14 @@ object TextTweaks {
   val italic = Tweak[W](x ⇒ x.setTypeface(x.getTypeface, Typeface.ITALIC))
   val boldItalic = Tweak[W](x ⇒ x.setTypeface(x.getTypeface, Typeface.BOLD_ITALIC))
 
-  val serif = Tweak[W](x ⇒ x.setTypeface(Typeface.SERIF, Option(x.getTypeface).map(_.getStyle).getOrElse(Typeface.NORMAL)))
-  val sans = Tweak[W](x ⇒ x.setTypeface(Typeface.SANS_SERIF, Option(x.getTypeface).map(_.getStyle).getOrElse(Typeface.NORMAL)))
-  val mono = Tweak[W](x ⇒ x.setTypeface(Typeface.MONOSPACE, Option(x.getTypeface).map(_.getStyle).getOrElse(Typeface.NORMAL)))
+  val serif = Tweak[W](x ⇒ x.setTypeface(Typeface.SERIF, typefaceStyle(x)))
+  val sans = Tweak[W](x ⇒ x.setTypeface(Typeface.SANS_SERIF, typefaceStyle(x)))
+  val mono = Tweak[W](x ⇒ x.setTypeface(Typeface.MONOSPACE, typefaceStyle(x)))
+  /** Set a typeface with the given name. For example:
+   *
+   *      w[TextView] <~ typeface("sans-serif-light")
+   */

```

Comment:
This would render a bit nicer in scaladoc popups:
```scala
/** Set a typeface with the given name
  *
  * Example:
  * {{
  * w[TextView] <~ typeface("sans-serif-light")
  * }}
  */